pkg/okteto/stream.go

```diff
@@ -73,3 +79,35 @@ func printPipelineLog(line string) bool {
 	oktetoLog.Println(pLog.Message)
 	return false
 }
+
+// DestroyAllLogs retrieves logs from the pipeline provided and prints them, returns error
+func (c *streamClient) DestroyAllLogs(ctx context.Context, namespace string) error {
+	streamURL := fmt.Sprintf(destroyAllUrlTempleate, Context().Name, Context().Namespace)
+	url, err := url.Parse(streamURL)
+	if err != nil {
+		return err
+	}
+	return stream.GetLogsFromURL(ctx, c.client, url.String(), printDestroyAllLog)
```

Comment:
what happens with reconnections? could we retry and keep iterating on the events?

Reply:
@teresaromero that's great!
will events be shown twice after a reconnection? or do we manage to only show events once?

Reply:
@pchico83  regarding reconnection, my mistake, what is implemented at the request for subscription is a retry for the initial connection, as this was a leading point to solve cloud issue...

a reconnection scenario where the connection is lost would require more changes. The logs come though the body stream, so we would need to detect if the scan of the body has stoped because a connection failure. the logs are pure text so there is no id or order we can identify though the stream... i think a reconnection strategy would need more refinement, as while the process continues, the reconnection time could lead us to miss some of the logs.

cc: @ifbyol

Reply:
Yeah, I wouldn't block this PR for that reconnection. As tere says, we don't have any kind of id, so it would require some extra work or refinement

Reply:
@teresaromero could you create an issue in the product backlog with these details? that would help us to prioritize it as needed